code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -363,7 +363,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 					continue
 				if(pref.gender == FEMALE && S.gender == MALE)
 					continue
-				if(!(bodytype in S.species_allowed))
+				if(!((bodytype in S.species_allowed) || (mob_species.name in S.species_allowed)))
```

Comment:
These two lists should be separate, the species name shouldn't be in the bodytype list

Reply:
Discussing this on discord has reached an impasse. I'd like @Alberyk and/or @Arrow768 to weigh in if possible.

Attempting to be as unbiased as possible:
Matt proposes that two different lists in the form of species_allowed and bodytype_allowed be used to clearly separate my proposed function into two different vars to avoid confusion should people visit body markings to add their own stuff.

My current changes allow species_allowed to use either the bodytype var in the species datum _or_ the species name in the same datum, condensing it's functionality down into just one list with minimal impact. These features are clearly documented.

I don't believe Matt's proposed changes are necessary in the pursuit of keeping things clear as to their function. Rather, I think my current changes are fine, since anyone who looks at species_allowed - without reading the commented documentation -, will immediately assume they use the species name, a function that I think should have been the case in the first place.

Reply:
Circumvented by Skull's request. Changes made.